docs/source/contents/4_draft_mapping.rst

```diff
@@ -2152,6 +2152,622 @@ Drop all values in mods:part.
+| dcterms:tableOfContents       | String Literal     | N/A   | A String Literal that represents constituent parts of a resource.                          |
++-------------------------------+--------------------+-------+--------------------------------------------------------------------------------------------+
+| dcterms:hasVersion            | URI                | N/A   | A String Literal that represents constituent parts of a resource.                          |
++-------------------------------+--------------------+-------+--------------------------------------------------------------------------------------------+
+| dbo:collection                | String Literal     | N/A   | A String Literal that represents the physical archival collection the resource belongs to. |
++-------------------------------+--------------------+-------+--------------------------------------------------------------------------------------------+
+| dbo:isPartOf                  | URI                | N/A   | A URI that represents the physical archival collection a resource belongs to.              |
++-------------------------------+--------------------+-------+--------------------------------------------------------------------------------------------+
+| opaque:memberOfArchivalSeries | URI/String Literal | N/A   | A URI or String Literal that indicates the archival series a resource belongs to.          |
++-------------------------------+--------------------+-------+--------------------------------------------------------------------------------------------+
+| opaque:sheetMusic_hostItem    | URI/String Literal | N/A   | A URI or String Literal that indicates the host item for the resource being described.     |
++-------------------------------+--------------------+-------+--------------------------------------------------------------------------------------------+
+| relators:[term]               | URI/String Literal | N/A   | Use with a role from MARC Code List of Relators role terms.                                |
+|                               |                    |       | Value is either text or URI from a controlled vocabulary (like                             |
+|                               |                    |       | Library of Congress Name Authority File).                                                  |
++-------------------------------+--------------------+-------+--------------------------------------------------------------------------------------------+
+
+relatedItem as simple structural metadata
+-----------------------------------------
+Use Case
+^^^^^^^^
+`relatedItem`, with and without attributes, is used in a number of collections to express structural relationships. These relationships will be handled/expressed by default behavior in our next-gen DAMS. In the case of `relatedItem/abstract`, the values present should be handled at the collection level.
+
+Justification
+^^^^^^^^^^^^^
```

Comment:
Perhaps include everything after your first sentence in the use case section above as your justification? I think it would be helpful to have some text in the justification for consistency's sake. The first sentence shows how we currently used the metadata and the rest says what we are going to do. relatedItem/abstract isn't structural metadata, so maybe just rename this category something like "elements we won't migrate" or split abstract out into something else?